generators/client/templates/angular/package.json.ejs

```diff
@@ -50,7 +50,7 @@
     "sockjs-client": "1.1.4",
     "webstomp-client": "1.2.0",
     <%_ } _%>
-    "tether": "1.4.3",
+    "popper.js": "1.14.1",
```

Comment:
I think we should remove this from dependency and let the library which manages it to take care of it unless its required as a peer dependency

Reply:
@deepu105, it's specified as peer dependency in bootstrap 4. ng-bootstrap doesn't specify it as peer or/ dependency.

Reply:
then lets remove it, we dont use JS from bootstrap

Reply:
I think we do load bootstrap JS as I see below module test to specify jquery loader for bootstrap JS:
  ```{ test: /bootstrap\/dist\/js\/umd\//, loader: 'imports-loader?jQuery=jquery' }```

In any case, I don't see any harm on explicitly loading peer dependency.

Reply:
I think its legacy code and we should remove it as well, we use ng bootstrap which doesn't use any bootstrap js

Reply:
If i remove popper.js dependency, then, during installation, I see below warning:
```warning " > bootstrap@4.0.0" has unmet peer dependency "popper.js@^1.12.9".```

Reply:
thats better than bundling unused code right

Reply:
I had look at ng-bootstrap code and indeed they wrote custom js for popovers. I will remove popper.js dependency as that's independent.